velox/docs/monthly-updates/march-2022.rst

```diff
@@ -65,4 +65,142 @@ Wilfong, Krishna Pai, Laith Sakka, Liang Tao, MJ Deng, Masha Basmanova, Orri
 Erling, Paula Lahera, Pedro Eugenio Rocha Pedreira, Pradeep Garigipati, Richard
 Barnes, Rui Mo, Sagar Mittal, Sergey Pershin, Simon Marlow, Siva Muthusamy,
 Sridhar Anumandla, Victor Zverovich, Wei He, Wenlei Xie, Yoav Helfman, Yuan
-Chao Chou, Zhenyuan Zhao, tanjialiang
\ No newline at end of file
+Chao Chou, Zhenyuan Zhao, tanjialiang
+
+
+********************
+Feature Of The Month
+********************
+
+
+Using vector readers/writers to simplify dealing with Velox vectors.
+--------------------------------------------------------------------
+
+
+Although vector readers and writers were created originally as part of the simple function's interface, they are highly
+convenient tools that can be used in isolation in aggregate and vector functions implementations, and in general
+anywhere we want to read or write vectors.
```

Comment:
nit: Perhaps, clarity that while these are easy to use and result is shorter simpler code, there is virtually no performance penalty for simplicity.